File: LeetCode/Easy/0485-max-consecutive-ones/0485-max-consecutive-ones.go
func findMaxConsecutiveOnes(nums []int) int {
    ret := 0
    temp := 0
    for i := 0; i < len(nums); i++ {
        if nums[i] == 1 {
            temp++
            if temp > ret {ret = temp}
        } else {
            temp = 0
        }
    }
    return ret
}

File: LeetCode/Easy/0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int ret = 0;
        int temp = 0;
        for(int i = 0; i < nums.length; ++i){
            if (nums[i] == 0) {
                ret = Math.max(ret, temp);
                temp = 0;
                continue;
            }
            ++temp;
        }
        return Math.max(ret, temp);
    }
}